lib/src/components/NxToggle/__tests__/NxToggleRTL.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import '@testing-library/jest-dom';
```

Comment:
And this doesn't mess up other tests if done in this file?

Reply:
> And this doesn't mess up other tests if done in this file?

Yup! .. and only if you reserve this file for RTL tests and not mix it with enzyme tests.